### PR TITLE
CuteMaze: Add recipe

### PR DESCRIPTION
**Pull request by win8linux**
[*CuteMaze*](https://gottcode.org/cutemaze/) is a top-down maze game built with Qt. The game has multiple algorithms for maze generation and maze theming, both with two of its built-in themes and custom themes.

Confirmed working on `x86_64`:
screenshot1
screenshot2
screenshot5
screenshot4

### CODE REVIEW

File: games-puzzle/cutemaze/cutemaze-1.3.3.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* Zooming out to see more of the maze
+* SVG themes (two themes come with the game)
+"
+HOMEPAGE="https://gottcode.org/cutemaze/"
+COPYRIGHT="2007-2024 Graeme Gott"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://gottcode.org/cutemaze/cutemaze-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5af6de035ce0686c791f0e49798ad85fbd8c270dcebb556701b07a47a634469c"
+SOURCE_DIR="cutemaze-$portVersion"
```
Comment (korli) [inline games-puzzle/cutemaze/cutemaze-1.3.3.recipe:20]: isn't this the default value for SOURCE_DIR?

**Review by win8linux — COMMENTED** [review]
Don't have access to a Haiku install at the moment, so can't test the recipe without `SOURCE_DIR` right now.

**Comment by Begasus** [comment]
As mentioned on IRC, there is a svg icon that can be converted into an rdef file, so best hold on untill this is added. :)

**Review by korli — APPROVED** [review]

**Comment by Begasus** [comment]
Thanks!

**Pull request merged**